custom/src/main/java/com/splunk/opentelemetry/resource/ResourceAttributesToSystemProperties.java

```diff
@@ -36,12 +42,43 @@ public void beforeByteBuddyAgent() {}
 
   @Override
   public void afterByteBuddyAgent() {
-    Attributes attributes = Resource.getDefault().getAttributes();
+    Attributes attributes = getResource().getAttributes();
     attributes.forEach(
         (k, v) -> {
           if (k.getType() == AttributeType.STRING) {
             System.setProperty("otel.resource." + k.getKey(), v.toString());
           }
         });
   }
+
+  private static Resource getResource() {
```

Comment:
Okay, I ended up writing this abomination to get the actual resource attributes used by tracer...

I'll create an issue in the SDK to do something about it.

Reply:
I don't have any great idea, the best I could think of right now is adding a `ResourceProvider` inside the auto-configuration module that constructs another `ConfigProperties` object and uses it to read the property. Anything else that came to my mind was too hacky to mention it 😅

Reply:
Thinking more on this...it would be good for the autoconfiguration to expose the auto-configured Resource. This is something that would be useful to have access to, outside of having it built into the SDK itself. 

Maybe a `getResource()` method directly on the OpenTelemetrySdkAutoConfiguration class? Create the issue. I think Anuraag might have an opinion.

Reply:
> it's unlikely we'll do much to support this use-case of turning resource attributes into system properties
Reading existing resource attributes seems like pretty general use case?

Reply:
well, depends.. Do we want to expose them on the SDK classes? Maybe, but that's something we have to carefully consider, because once we expose them, we can't un-expose them.